src/substitutions.ts

```diff
@@ -29,9 +29,9 @@ const PILL_REGEX = /<a href="https:\/\/matrix\.to\/#\/(#|@|\+)([^"]+)">([^<]+)<\
  * Will return the emoji's name within ':'.
  * @param name The emoji's name.
  */
-export function getFallbackForMissingEmoji(name): string {
-    return `:${name}:`;
-}
+export const getFallbackForMissingEmoji = (name): string => (
```

Comment:
Again feels a little weird to enforce arrow functions wholesale. Makes more sense for nested functions, but not exported ones.

Reply:
I didn't enable this but the automatic converter for tslint to eslint configs added this rule – we can remove it.

Usually, I try to avoid `function` in vanilla JS to avoid collisions in the global namespace.
I guess TypeScript makes everything a module anyway.